lib/ansible/module_utils/basic.py

```diff
@@ -294,6 +294,13 @@ def get_distribution():
     return distribution
 
 
+def missing_required_lib(library):
+
+    hostname = platform.node().split('.')[0]
+    return "Cannot run as we are missing required library (%s) on %s's python %s."
+    "Please read module documentation and install in the appropriate location." % (library, hostname, sys.executable)
```

Comment:
This won't work.  You will only get the first line.  You'll need to wrap in parens.

```
    return ("Cannot run as we are missing required library (%s) on %s's python %s. "
            "Please read module documentation and install in the appropriate location." % (library, hostname, sys.executable))
```

A space also needs to be added after `python %s.`